.eslintrc.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-  extends: ['@mate-academy/eslint-config', 'plugin:cypress/recommended', 'plugin:chai-friendly/recommended'],
-  plugins: ['cypress', 'chai-friendly'],
+  extends: ['@mate-academy/eslint-config', 'plugin:cypress/recommended'],
```

Comment:
The 'chai-friendly' plugin is missing from the 'plugins' array. Please add it back to ensure adherence to the recommended code style linting. 😊